crypto/fipsmodule/sha/asm/sha512-x86_64.pl

```diff
@@ -129,9 +129,11 @@
 # versions, but BoringSSL is intended to be used with pre-generated perlasm
 # output, so this isn't useful anyway.
 #
-# TODO(davidben): Enable AVX2 code after testing by setting $avx to 2. Is it
-# necessary to disable AVX2 code when SHA Extensions code is disabled? Upstream
-# did not tie them together until after $shaext was added.
+# This file also has an AVX2 implementation, controlled by setting $avx to 2.
+# For now, we intentionally disable it. While it gives a 13-16% perf boost, the
+# CFI annotations are wrong. It allocates stack in a loop and should be
+# rewritten to avoid this.
+$avx = 1;
```

Comment:
`avx=1` is done twice (here, and in line 138). Also I think we can remove the `set to zero if compiling for 1.0.1` comment because it came from OpenSSL and it's not relevant for aws-lc

Reply:
Done. Thanks.